src/coreclr/jit/importer.cpp

```diff
@@ -11721,111 +11716,108 @@ void Compiler::impImportBlockCode(BasicBlock* block)
         if (opts.compDbgInfo)
 #endif
         {
-            if (!compIsForInlining())
-            {
-                nxtStmtOffs =
-                    (nxtStmtIndex < info.compStmtOffsetsCount) ? info.compStmtOffsets[nxtStmtIndex] : BAD_IL_OFFSET;
+            nxtStmtOffs =
```

Comment:
FWIW I've always though the logic in this bit of code was convoluted. But it looks like all that's happened here is that you removed the exclusion for inlinees?

Reply:
Yes, that's the only change here (it's easier to see if you check "Hide whitespace" under the review settings).